internal/namespaces/instance/v1/custom_server_create_test.go

```diff
@@ -43,33 +54,154 @@ func Test_CreateServer(t *testing.T) {
 
 		t.Run("With name", core.Test(&core.TestConfig{
 			Commands:  GetCommands(),
-			Cmd:       "scw instance server create image=ubuntu-bionic name=yo",
+			Cmd:       "scw instance server create image=ubuntu_bionic name=yo",
 			AfterFunc: deleteServerAfterFunc,
 			Check: func(t *testing.T, ctx *core.CheckFuncCtx) {
 				assert.Equal(t, "yo", ctx.Result.(*instance.Server).Name)
 			},
 		}))
 
+		t.Run("With start", core.Test(&core.TestConfig{
+			Commands:  GetCommands(),
+			Cmd:       "scw instance server create image=ubuntu_bionic start -w",
+			AfterFunc: deleteServerAfterFunc,
+			Check: func(t *testing.T, ctx *core.CheckFuncCtx) {
+				assert.Equal(t, instance.ServerStateRunning, ctx.Result.(*instance.Server).State)
+			},
+		}))
+
 		t.Run("With bootscript", core.Test(&core.TestConfig{
 			Commands:  GetCommands(),
-			Cmd:       "scw instance server create image=ubuntu-bionic bootscript-id=eb760e3c-30d8-49a3-b3ad-ad10c3aa440b",
+			Cmd:       "scw instance server create image=ubuntu_bionic bootscript-id=eb760e3c-30d8-49a3-b3ad-ad10c3aa440b",
 			AfterFunc: deleteServerAfterFunc,
 			Check: func(t *testing.T, ctx *core.CheckFuncCtx) {
```

Comment:
Why not using [TestCheckEqual](https://github.com/scaleway/scaleway-cli/blob/v2/internal/core/testing.go#L313)

Reply:
Because this is typed

Reply:
So could do something with TestCheckEqual, remove / improve it

Reply:
`TestCheckEqual` is able to compare value between the result and an external object added in the Meta from the `BeforeFunc`, that's its only advantage IMO

Reply:
I agree typed tests make more sense. Let see if TestCheckEqual is useful in some future case.